modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -235,21 +230,24 @@ I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Support-Role for th
 ? Multiple availability zones (optional): No <7>
 ? AWS region: us-east-1
 ? PrivateLink cluster (optional): No
-? Install into an existing VPC (optional): No
+? Install into an existing VPC (optional): Yes <8>
 ? Select availability zones (optional): No
-? Enable Customer Managed key (optional): No <8>
+? Enable Customer Managed key (optional): No <9>
 ? Compute nodes instance type (optional):
 ? Enable autoscaling (optional): No
+? Additional Compute Security Group IDs (optional): <10>
```

Comment:
The question is later than compute nodes
```
lixue@Xue-Lis-MacBook-Pro ~ % rosa create cluster -c xuelisg --channel-group candidate -i
I: Interactive mode enabled.
Any optional fields can be left empty and a default will be selected.
? Cluster name: xuelisg
? Deploy cluster with Hosted Control Plane: No
? Create cluster admin user: No
? Deploy cluster using AWS STS: No
W: In a future release STS will be the default mode.
W: --sts flag won't be necessary if you wish to use STS.
W: --non-sts/--mint-mode flag will be necessary if you do not wish to use STS.
? OpenShift version: 4.14.1
? Configure the use of IMDSv2 for ec2 instances optional/required: optional
? Tags (optional): 
? Multiple availability zones: No
? AWS region: us-west-2
? PrivateLink cluster: No
? Private cluster: No
? Machine CIDR: 10.0.0.0/16
? Service CIDR: 172.30.0.0/16
? Pod CIDR: 10.128.0.0/14
? Install into an existing VPC: Yes
W: The following subnets were excluded because they belong to a VPC that is managed by Red Hat: [subnet-006e6d54d83334a56, subnet-025969a649813eb8e, subnet-02def134f473db609, subnet-034f6721ce8d888bd, subnet-040fa13bb054fe0d4, subnet-04bc34e4a0c2de7b0, subnet-07c655126d042a2e5, subnet-09ed7ea79994d7651, subnet-09f8b6882c29d0e5e, subnet-0a4c8f2e61d71e4f5, subnet-0ba8d1ba763971c48, subnet-0e15d163e0bcb3f9f]
? Subnet IDs (optional): subnet-0c658c880dff1afbd ('rosa-sts-ad-rosa-sts-ad-main-public-a','vpc-0ef3d4499a7c49021','us-west-2a', Owner ID: '301721915996'), subnet-0e2b22f650a77458c ('rosa-sts-ad-rosa-sts-ad-main-private-a','vpc-0ef3d4499a7c49021','us-west-2a', Owner ID: '301721915996')
? Enable Customer Managed key: No
? Compute nodes instance type: m5n.metal
? Enable autoscaling: No
? Compute nodes: 2
? Worker machine pool labels (optional): 
? Additional Security Group IDs (optional):  [Use arrows to move, space to select, <right> to all, <left> to none, type to filter, ? for more help]
> [ ]  sg-0d42fb2fb31c362c2 ('rosa-sts-ad-main-3')
  [ ]  sg-0d1dafbaecf5bd77e ('rosa-sts-ad-main-2')
  [ ]  sg-0df7548707c641882 ('rosa-sts-ad-main-1')
  [ ]  sg-0ecd5aac46407fad2 ('rosa-sts-ad-main-0')
```

Reply:
Thankyou! Updating.

Reply:
Also correcting to remove Compute from Additional *Compute* Security Group IDs based on the above.